godot-core/src/builtin/string/string_macros.rs

```diff
@@ -474,3 +474,15 @@ macro_rules! impl_shared_string_api {
         }
     };
 }
+
+#[macro_export]
+macro_rules! godot_str {
+    ($fmt:literal $(, $args:expr)* $(,)?) => {
+        $crate::global::str(&[
+            $crate::builtin::Variant::from(
+                format!($fmt $(, $args)*)
+            )
+        ])
+    };
+}
```

Comment:
It would be good to add this at the end of the file https://github.com/godot-rust/gdext/blob/master/godot-core/src/global/print.rs, since it's so similar to the others.

Could you also add a small documentation, maybe even with an doctest example that shows just 2-3 things being concatenated?